GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
 of exploration.
 
 
+Managing inventory and crafting
+-------------------------------
+
+All items belonging to your team are stored either in the character's personal 
+equipment outfit or in the shared inventory stash that is accessible by every 
+team member. 
+Items that are in the personal equipment outfit can be accessed by or 
+are beneficial for this particular character only. Stashed items can be 
+accessed by the team members (unless the character is not calm enough 
+or injured). But no one has a direct advantage from having the items 
```

Comment:
How about "direct stats advantage" or "direct benefit to skill level"?

Reply:
What is it?

Reply:
What is what? Skill?

Reply:
Yes, what do you mean by these terms? 
"direct stats advantage", "direct benefit to skill level".

Reply:
I mean that some items, for as long as worn, make skills (as seen in the # menu) of the character higher.

Did you mean something else by your "direct advantage"? I suppose also the ability to use an item in melee, if applicable? Anything else?

Reply:
Oh, I see. 
Yes, exactly this. That items stored in the shared inventory do not give bonuses to the team. The statement is a bit vague, probably, but very simple.